explainers/security-privacy-questionnaire-post-task.md

```diff
@@ -0,0 +1,154 @@
+# `scheduler.postTask()`: Security and Privacy Questionnaire Answers
+
+The following are the answers to the W3C TAG's
+[security and privacy self-review questionnaire](https://w3ctag.github.io/security-questionnaire/).
+
+> 01. What information might this feature expose to Web sites or other parties,
+>     and for what purposes is that exposure necessary?
+
+Tasks scheduled with this API have an associated priority, which is provided by
+consumers of the API. This priority could be obtained in JavaScript either by
+having access to the associated `TaskSignal`, or by monkey-patching the
+`scheduler` and intercepting calls to `postTask()`. The priority is used by the
+UA for scheduling decisions, and is fundamental to a prioritized scheduling
+API.
+
+Tasks also have an optional delay value &mdash; analogous to `setTimeout()`'s
+timeout value. This could similarly be obtained in JavaScript by
+monkey-patching the API. The delay value is required to schedule delayed tasks.
+
+In the context of [Third-Party Tracking](https://w3ctag.github.io/security-questionnaire/#third-party-tracking),
```

Comment:
This probably needs some more expansion. How can third parties monitor scheduled tasks? Recall that "third party" = a different iframe with no sync access (because a third party is cross-site by definition), so it can't monkeypatch.

Reply:
Upon reading further, maybe this is already covered by the "other origins" question.

Reply:
See previous comment. Yeah the question here was "what can 3P learn about 1P's scheduled tasks when running in the same context?", which seems unnecessary based on your other comment.